src/screens/WebsocketResearchScreen/SampleChatInfoScreen.tsx

```diff
@@ -51,14 +62,14 @@ export const styles = StyleSheet.create({
     alignSelf: 'center',
     bottom: 5
   },
-  countUser: {
-    fontFamily: fonts.inter[600],
+  countUser: (from) => ({
     fontSize: normalizeFontSize(14),
     lineHeight: normalizeFontSize(16.94),
-    color: colors.holytosca,
-    marginLeft: 20,
-    marginBottom: 4
-  },
+    color: from === SIGNED ? colors.darkBlue : colors.holytosca,
+    marginLeft: dimen.normalizeDimen(20),
+    marginBottom: dimen.normalizeDimen(4),
+    fontWeight: 'bold'
+  }),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `countUser` style has been updated to be a function that takes an argument `from`. Make sure this change doesn't break any existing usage of `countUser` style elsewhere in the codebase.



<!-- This is an auto-generated comment by OSS CodeRabbit -->